python/sqlflow_submitter/xgboost/train.py

```diff
@@ -33,8 +33,12 @@ def train(datasource, select, model_params, train_params, feature_field_meta, la
     feature_spec = {k['name']: k for k in feature_field_meta}
 
     dtrain = xgb_dataset(conn, 'train.txt', select, feature_column_name, label_name, feature_spec)
-    # FIXME(weiguoz): bring dtest back when VALIDATE clause is ready
-    # dtest = xgb_dataset('test.txt', '''{{.ValidationSelect}}''')
+    watchlist = [(dtrain, "train")] 
+    if len(validation_select.strip()) > 0:
+        dvalidate = xgb_dataset(conn, 'validate.txt', validation_select, feature_column_name, label_name, feature_spec)
+        watchlist.append((dvalidate, "validate"))
 
-    bst = xgb.train(model_params, dtrain, **train_params)
-    bst.save_model("my_model")
\ No newline at end of file
+    re = dict()
+    bst = xgb.train(model_params, dtrain, **train_params, evals=watchlist, evals_result=re)
+    bst.save_model("my_model")
+    print(re)
```

Comment:
maybe add some message: `print("Validation result: %s" % re)`?

Reply:
This will be fixed in the next `polish code` pr.